src/NetAnalyzers/Core/Microsoft.CodeQuality.Analyzers/MicrosoftCodeQualityAnalyzersResources.resx

```diff
@@ -1263,4 +1263,12 @@
   <data name="OperatorOverloadsHaveNamedAlternatesCodeFixTitle" xml:space="preserve">
     <value>Add operator overload named alternate</value>
   </data>
-</root>
+  <data name="AvoidMultipleEnumerationsMessage" xml:space="preserve">
+    <value>Whenever an 'IEnumerable' collection is used, it will be enumerated. Try convert it to.</value>
```

Comment:
Cut off text?

Reply:
Yup, I will polish this part in the following commit